akka/src/main/scala/cakesolutions/kafka/akka/ConsumerRecords.scala

```diff
@@ -0,0 +1,167 @@
+package cakesolutions.kafka.akka
+
+import scala.reflect.runtime.universe.{TypeTag, typeTag}
+import cakesolutions.kafka.KafkaProducerRecord
+import org.apache.kafka.clients.consumer.{ConsumerRecord => JConsumerRecord, ConsumerRecords => JConsumerRecords}
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.TopicPartition
+
+import scala.collection.JavaConverters._
+
+/**
+  * Helper functions for [[ConsumerRecords]].
+  */
+object ConsumerRecords {
+  type Pair[Key, Value] = (Option[Key], Value)
+  type Partition = (String, Int)
+
+  /**
+    * Create consumer records for a single partition from values only.
+    */
+  def fromValues[Key >: Null : TypeTag, Value: TypeTag](partition: Partition, values: Seq[Value]): ConsumerRecords[Key, Value] =
+    fromPairs(partition, values.map(None -> _))
+
+  /**
+    * Create consumer records for a single partition from key-value pairs.
+    */
+  def fromPairs[Key >: Null : TypeTag, Value: TypeTag](partition: Partition, pairs: Seq[Pair[Key, Value]]): ConsumerRecords[Key, Value] =
+    fromMap(Map(partition -> pairs))
+
+  /**
+    * Create consumer records from a map of partitions and key-value pairs.
+    */
+  def fromMap[Key >: Null : TypeTag, Value: TypeTag](values: Map[Partition, Seq[Pair[Key, Value]]]): ConsumerRecords[Key, Value] = {
+    def createConsumerRecords(topic: String, partition: Int, pairs: Seq[Pair[Key, Value]]) =
+      pairs.zipWithIndex.map {
```

Comment:
It's mostly useful for testing purposes. For example, you might want to test that an actor correctly handles `ConsumerRecords` properly without involving a real Kafka consumer. I'll add more documentation for this.